src/util.js

```diff
@@ -601,6 +601,7 @@ const util = {
     // update error message
     try {
       error.message = message + ': ' + error.message;
+      error.name = 'Error';
```

Comment:
Why/where is this change needed?

Reply:
Without this, the high-level error looks a bit strange, as you get e.g. `UnsupportedError: Error decrypting message: Unsupported packet version ...` or w/e. So it's a bit redundant.
In a separate PR I'd like to change the error messages to be more like `Error: Decrypting message failed: Unsupported packet version ...` for example.